twilight-http/src/request/multipart.rs

```diff
@@ -128,10 +126,9 @@ impl Default for Form {
 /// Generate a random boundary that is 15 characters long.
 pub fn random_boundary() -> [u8; 15] {
     let mut boundary = [0; 15];
-    let mut rng = rand::thread_rng();
 
     for value in &mut boundary {
-        *value = rng.sample(Alphanumeric);
+        *value = fastrand::alphanumeric() as u8;
     }
 
     boundary
```

Comment:
That won't work, boundaries have to be ASCII (Unlike some other parameter values, the values of the charset parameter are NOT case sensitive. The default character set, which must be assumed in the absence of a charset parameter, is US-ASCII)

Reply:
Looking further into this, I've found conflicting resources on whether the boundary should be randomized or not, and notably I've not found any such mention in an RFC. Do you happen to know? Otherwise it seems silly to randomize the boundary at all.

Reply:
Let's leave it at this for now. The issue is tracked in #2278